.gitignore

```diff
@@ -197,6 +197,7 @@ tvm_t.*
 .pytest_cache
 .local
 cmake-build-debug
+cmake-build*
```

Comment:
your pr should not be related with this .gitignore

Reply:
Should modify `.gitignore` in another PR ?

Reply:
Yes, if you think it is a must